docs/b2b-edition/specs/storefront/storefront/company.yaml

```diff
@@ -2802,7 +2804,7 @@ paths:
           name: role
           description: 'User role, 0： "Admin" 1： "Senior Buyer" 2： "Junior Buyer" 3： "Sales Rep" 4： "UNKNOWN"'
           required: true
-      description: Check if a user email can be used for a company; user info and available status will be in the reponse(backend)
+      description: Check if a user email is valid for a company; user information and available status are back-end responses.
       tags:
         - Company
       x-internal: true
```

Comment:
@bc-traciporter - These items marked as `x-internal` are supposed to be only visible to internal team as they are used by the b2b edition control panel project and not used externally.